### ISSUE TITLE
Asymmetric blood pressure between arms

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10473]
**Preferred term label:**

**Synonyms**
blood pressure asymmetry

**Definition (free text, please give PubMed ID)**
PMID: 36351705 Blood pressure readings were only recorded for the upper extremities in the database. Difference in systolic blood 
pressure in the upper extremities was categorized at 10 mmHg intervals. A difference in systolic blood pressure of 20 mmHg maximized sensitivity and specificity between TAK and GCA and was selected for further analysis

PMID: 37525862 In around 85% of patients, limb signs or symptoms are one of the most typical TA presentations. These include weak or missing pulses, claudication in the affected limbs, and **differences in blood pressure between the right and left arms**.

**Parent term (use hpo.jax.org/app)**
Asymmetric blood pressure between limbs (See issue #10472 )

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Takayasu arteritis  ORPHA:3287

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6000945
- label: Asymmetric blood pressure between arms
- parents: [Abnormal systemic blood pressure (HP:0030972)](https://hpo.jax.org/app/browse/term/HP:0030972)
- pmid: PMID:36351705|PMID:37525862
- def: A substantial difference in blood pressure (e.g., 20 mmHg) between the right and left arms.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10473
- synonyms:
